tests/cases/standalone/common/create/create_metric_table.result

```diff
@@ -71,6 +71,27 @@ DESC TABLE phy;
 | job        | String               | PRI | YES  |         | TAG           |
 +------------+----------------------+-----+------+---------+---------------+
 
+SHOW CREATE TABLE phy;
```

Comment:
Why can’t see the `__table_id` and `__tsid` columns?

Reply:
The `__table_id` and `__tsid` are internal columns, and these columns and column options will be generated via the metric engine.